src/Synths/HookTargetSynth.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.0;
+
+import {IHookTarget} from "../interfaces/IHookTarget.sol";
+import {IEVault} from "../EVault/IEVault.sol";
+
+/// @title HookTargetSynth
+/// @custom:security-contact security@euler.xyz
+/// @author Euler Labs (https://www.eulerlabs.com/)
+/// @notice HookTargetSynth is designed to block unnecessary operations and enforce deposits only by the asset contract.
```

Comment:
We should document somewhere *which* hooks this is expected to be installed for.